js/collapse.js

```diff
@@ -154,6 +149,21 @@
       .attr('aria-expanded', isOpen)
   }
 
+  Collapse.prototype.updateTriggers = function ($currentTrigger) {
+    this.$allTriggers  = $('[data-toggle="collapse"][href="#' + this.$element[0].id + '"],'
```

Comment:
No, because `[data-toggle="collapse"]` is used as an identifier to find all triggers. `options.trigger` is set to point to the actual trigger that triggered the collapse.